data/strings/strings.txt

```diff
@@ -24063,6 +24607,7 @@
     ja = 自動
     ko = 자동
     lt = Automatinis
+    lv = Kad ir zems uzlādes līmenis
     mr = स्वयंचलित
     nb = Automatisk
```

Comment:
I wonder why English translation is different from all other languages? @pastk @biodranik

Reply:
I can't comment on this one.